Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/ImportBeforeTargetsTests.cs

```diff
@@ -43,6 +43,7 @@ public class ImportBeforeTargetsTests
         [TestInitialize]
```

Comment:
Not sure why, but not all tests passed when I used AssemblyInitialize, hence TestInitialize everywhere